client/webserver/site/src/js/forms.ts

```diff
@@ -1027,7 +1027,7 @@ export class AccelerateOrderForm {
     this.currencyUnit = preAccelerate.suggestedRange.yUnit
     page.accelerateAvgFeeRate.textContent = `${preAccelerate.swapRate} ${preAccelerate.suggestedRange.yUnit}`
     page.accelerateCurrentFeeRate.textContent = `${preAccelerate.suggestedRate} ${preAccelerate.suggestedRange.yUnit}`
-    OrderUtil.setOptionTemplates(page)
+    OrderUtil.setRangeOptionTempalate(page.accelerationRangeOptTmpl)
```

Comment:
If the `accelerationRangeOptTmpl` is different than the `rangeOptTmpl`, then please make a new `class`. Maybe it can inherit `XYRangeHandler`.

Reply:
It wasn't different. I've done some refactoring.. all the templates are now in a template in `forms.tmpl`, and included in `markets.tmpl` and `order.tmpl`. I think it's much cleaner now.